### PR TITLE
First prototype of auto labeler

### PR DESCRIPTION
**Pull request by shrutix**
### Summary

Prototype of auto labeler for articles and publications. Uses transformers and BERT architecture (fine-tuned) with PyTorch to classify wikipedia articles. 
Current classification accuracy: `11%`

### Related Issues

- ~~Resolves~~ Starts resolving Issue # [1](https://github.com/nasa/PeTaL-labeller/issues/1)

### Backwards incompatibilities

Does this PR cause any incompatibilites with previous versions? Otherwise, None.

### New Dependencies

Does this PR require the use of a new dependency? Otherwise, None.

### CODE REVIEW

**Comment by pjuangph** [comment]
1. Shruthi in the demo, i remember you had some technical problems with the code. Have you addressed that? 

2. You reference both tensorflow and pytorch? do you actually use tensorflow? Can you remove unused references? Also remove lengthy print statements.

**Comment by shrutix** [comment]
@pjuangph turns out those were some issues with the colab gpu, it's running smoothly now
I removed unecessary print statements and I think I use tensorflow in the beginning to check that the gpu is available.

**Review by pjuangph — APPROVED** [review]
